twd.htm

```diff
@@ -206252,16 +206217,9 @@ <h3><a id="Year1997">1997</a></h3>
 6 x Conditioning
 5 x Deflection
 4 x Computer Hacking
-4 x Manouever
+4 x Apportation
```

Comment:
I know the SL import of the deck uses Apportation here, but given the lack of THA in the crypt, and the generally defensive nature of the combat package, I believe ''Fake Out' is the more likely card.

Also, this change should be noted somewhere in a comment on the deck.